exporter/oci/export.go

```diff
@@ -144,9 +139,7 @@ func (e *imageExporterInstance) Export(ctx context.Context, src exporter.Source)
 	resp := make(map[string]string)
 
 	if n, ok := src.Metadata["image.name"]; e.name == "*" && ok {
-		if e.name, err = normalize(string(n)); err != nil {
-			return nil, err
-		}
+		e.name = string(n)
```

Comment:
Why did this change?

Reply:
@tonistiigi This looks like my mistake. I was testing something internally and then forgot to revert it and made a commit. I will revert it.